src/states/BattleResolveState.py

```diff
@@ -60,7 +61,7 @@ def update(self, dt, events):
             })
 
     def resolveCardEffect(self, effect, effectOwner):
-        if effect.type == EffectType.ATTACK:
+        if effect.type == "attack":
```

Comment:
@Patharapol1573 Why change these from ENUM value to string value? The former handle with ENUM seem to be more reliable configurations